src/components/modeler/XMLManager.js

```diff
@@ -47,6 +48,7 @@ export default class XMLManager {
           }
           return true;
         });
+        element.laneSets[0].lanes = element.laneSets[0].lanes.filter(node => !!node.flowNodeRef);
```

Comment:
There are two problems:

- This code is inside `if (element.$type === 'bpmn:Process' && element.flowElements) {` but should be outside because LaneSets are at the same level of Process Elements.
- Also take in could that the diagram could have more than one pool, so the LaneSets could be more than one.